sidebars.ts

```diff
@@ -300,75 +298,82 @@ const sidebars: SidebarsConfig = {
         id: 'platform/howto/list-monitoring',
       },
       items: [
-        'platform/howto/monitoring-services',
-        'platform/concepts/logs-metrics-alerts',
-        'tutorials/anomaly-detection',
         {
           type: 'category',
-          label: 'Amazon CloudWatch and Aiven',
+          label: 'Metrics integrations',
```

Comment:
Wdyt about including logging in this? Something like "Metric and log integrations".